src/components/InventoryV2/DownloadReportModal.tsx

```diff
@@ -0,0 +1,20 @@
+import ExportCsvModal from 'src/components/common/ExportCsvModal';
+import NurseryInventoryService, { SearchInventoryParams } from 'src/services/NurseryInventoryService';
+
+interface DownloadReportModalProps {
+  reportData?: SearchInventoryParams;
+  open: boolean;
+  onClose: () => void;
+}
+
+export default function DownloadReportModal(props: DownloadReportModalProps): JSX.Element {
+  const { reportData, open, onClose } = props;
+
+  const onExport = async () => {
+    if (reportData) {
```

Comment:
this check should probably be in the calling code.. the purpose of this modal is to download csv and should be ok to require the reportData prop